### ISSUE TITLE
Idea: OpenModelica Language Server Protocol

### ISSUE THREAD

**Issue by OpenModelica-TracImporter** [OpenModelica/OpenModelica#4458]

**Comment by thorade** [comment]
Originally suggested here
https://github.com/THM-MoTE/mope-server/issues/3
https://trac.openmodelica.org/OpenModelica/ticket/4458 

By now, OpenModelica has a language server, so this ticket here can be closed
https://github.com/OpenModelica/modelica-language-server

Also adding these two related links:
https://github.com/prettier/prettier/discussions/16901  
https://github.com/OpenModelica/tree-sitter-modelica/issues/25

**Comment by AnHeuermann** [comment]
Yeah, we "have" a language server. But it can't do a lot of stuff besides goto-definition and function outlines. But if someone wants to improve on it we accept pull requests.

**Comment by AnHeuermann** [comment]
OMEdit got a Modelica LSP server in https://github.com/OpenModelica/OpenModelica/pull/15925.
